diracx-logic/src/diracx/logic/auth/token.py

```diff
@@ -394,3 +418,8 @@ async def get_authorization_flow(auth_db: AuthDB, code: str, max_validity: int):
         raise AuthorizationError("Code was already used")
 
     raise AuthorizationError("Bad state in authorization flow")
+
+
+def generate_pilot_secret():
+    # Can change with time
+    return token_hex(16)
```

Comment:
https://docs.python.org/3/library/secrets.html#how-many-bytes-should-tokens-use
suggests to use at least 32 bytes - this can't hurt

```suggestion
    return token_hex(32)
```